product_classification_test_data/mixxmix/nodes.txt

```diff
@@ -1 +1 @@
8,"display":"","strikethrough":"none","left":955,"top":3084,"right":955,"visibility":""},"586":{"bottom":3098,"display":"","strikethrough":"none","left":470,"top":3098,"right":1440,"visibility":""},"587":{"bottom":3098,"display":"","strikethrough":"none","left":470,"top":3098,"right":1440,"visibility":""},"588":{"bottom":3098,"display":"","strikethrough":"none","left":470,"top":3098,"right":1440,"visibility":""},"589":{"bottom":3098,"display":"","strikethrough":"none","left":470,"top":3098,"right":1440,"visibility":""},"590":{"bottom":8738,"display":"","strikethrough":"none","left":470,"top":3098,"right":1440,"visibility":""},"591":{"bottom":4480,"display":"","strikethrough":"none","left":494.25,"top":3098,"right":1415.75,"visibility":""},"592":{"bottom":3098,"display":"","strikethrough":"none","left":1415.75,"top":3098,"right":1415.75,"visibility":""},"593":{"bottom":4494,"display":"","strikethrough":"none","left":955,"top":4480,"right":955,"visibility":""},"594":{"bottom":4508,"display":"","strikethrough":"none","left":955,"top":4494,"right":955,"visibility":""},"595":{"bottom":5890,"display":"","strikethrough":"none","left":494.25,"top":4508,"right":1415.75,"visibility":""},"596":{"bottom":4508,"display":"","strikethrough":"none","left":1415.75,"top":4508,"right":1415.75,"visibility":""},"597":{"bottom":5904,"display":"","strikethrough":"none","left":955,"top":5890,"right":955,"visibility":""},"598":{"bottom":5918,"display":"","strikethrough":"none","left":955,"top":5904,"right":955,"visibility":""},"599":{"bottom":7300,"display":"","strikethrough":"none","left":494.25,"top":5918,"right":1415.75,"visibility":""},"600":{"bottom":5918,"display":"","strikethrough":"none","left":1415.75,"top":5918,"right":1415.75,"visibility":""},"601":{"bottom":7314,"display":"","strikethrough":"none","left":955,"top":7300,"right":955,"visibility":""},"602":{"bottom":7328,"display":"","strikethrough":"none","left":955,"top":7314,"right":955,"visibility":""},"603":{"bottom":8710,"display":"","strikethrough":"none","left":494.25,"top":7328,"right":1415.75,"visibility":""},"604":{"bottom":7328,"display":"","strikethrough":"none","left":1415.75,"top":7328,"right":1415.75,"visibility":""},"605":{"bottom":8724,"display":"","strikethrough":"none","left":955,"top":8710,"right":955,"visibility":""},"606":{"bottom":8738,"display":"","strikethrough":"none","left":955,"top":8724,"right":955,"visibility":""},"607":{"bottom":8738,"display":"","strikethrough":"none","left":470,"top":8738,"right":1440,"visibility":""},"608":{"bottom":8738,"display":"","strikethrough":"none","left":470,"top":8738,"right":1440,"visibility":""},"609":{"bottom":8738,"display":"","strikethrough":"none","left":470,"top":8738,"right":1440,"visibility":""},"610":{"bottom":8738,"display":"","strikethrough":"none","left":470,"top":8738,"right":1440,"visibility":""},"611":{"bottom":8752,"display":"","strikethrough":"none","left":470,"top":8738,"right":1440,"visibility":""},"612":{"bottom":8752,"display":""
```

Comment:
It occurs to me that our nod eindexing could be thrown off if JS mutates the DOM while the page is loaded in Safari. I don't have any immediate ideas for detecting or mitigating this, short of disabling JS. Do you?